src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+
+> Using a microphone (even a common corded headset) can help amplify voices and minimize distracting unwanted sounds.
+
+### Reduce background music
+Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
+
+> Consider putting your background music at the volume of a whisper.
+
+### Provide text with multimedia
+As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
+
+> Where there’s media, make sure there’s also text.
+
+### Do not have any flashing content
+Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes.
+
+> The Trace Center’s [Photosensitive Epilepsy Analysis Tool (PEAT)](https://trace.umd.edu/peat/) is a free, downloadable resource to identify seizure risks in  web content and software.
+
+## Advocate for Accessibility
+
+### Seek accessibility features
+Actively search for “accessibility” in the tools you use every day — from survey creation sites to website design platforms. Understand what you can do to make sure your output is inclusive.
+
+<figure role="figure" aria-label="Search for accessibility tips in your favorite systems like this Typeform How To article.">
+	<img alt="Help Center page from Typeform, an online survey creation service, regarding how to make accessible typeforms." src="/img/posts/2021-06-14-Everyday_Accessibility/Accessible_Typeforms.png" />
+	<figcaption>Search for accessibility tips in your favorite systems like this Typeform "How To" article.</figcaption>
+</figure>
+
+### Ask about accessibility
+If a system doesn’t offer accessibility features (which is unfortunately very common), reach out to the company about their accessibility policy and roadmaps via the customer support channels. Also ask about it internally where you work and attend school. Don’t leave this to just your disabled colleagues to take on.
+
+> Customer pressure can help move accessibility to the front of a company’s roadmap.
+
+### Seek to learn more 
+There’s a lot more to learn and the more allies who know about accessible and inclusive digital practices, the more we can make a big difference for who can participate in our ever-growing digital spaces.
+
+> You can start your journey with [The A11Y Project’s Resources Page](A11yProject.com/Resources).
```

Comment:
🧡

Reply:
This was literally my idea to put even before I thought "Hey, maybe they might want this article!" LOL